### DISCUSSION TITLE
codeblock环境报错

### DISCUSSION THREAD

**Discussion by 2BeKindPLZ** [sjtug/SJTUThesis#1068] [提问] [answered]
Texstuidio在编译之后报了如图中所示错误，是在floats.tex文件中的。
Overleaf中没有报错，成功编译了，所以应该是我本地的配置有问题？
似乎是codeblock环境没有定义导致的吗？附件是错误截图以及对应的log文件。
运行环境是TeX Live，Win10系统。
![float](https://github.com/user-attachments/assets/e75bd124-49e6-4210-a225-a7e5375e967e)

**Comment by AlexaraWu** [comment, marked as answer]
看起来你在用户目录 `%userprofile%\texmf` 里安装了一个 v2.0.3 旧版的 `sjtuex`。

```log
...
(c:/Users/rickey/texmf/tex/latex/sjtutex/sjtuthesis.cls
...
Document Class: sjtuthesis 2023/09/25 v2.0.3 Thesis template for Shanghai Jiao 
Tong University
...
```

你把旧版的删掉，再下载最新的 v2.1.5 模板试试。

**Reply by 2BeKindPLZ** [reply]
谢谢哈！！！我上周四下载了最新的模板，但是好像和之前下载的搞混了，长期相信自己用的是最新模板=_=
